invenio_app_ils/documents/schemas/documents/document-v1.0.0.json

```diff
@@ -590,21 +590,18 @@
           }
         },
         "multipart_monograph": {
-          "items": {
-            "properties": {
-              "pid": {
-                "type": "string"
-              },
-              "pid_type": {
-                "type": "string"
-              },
-              "volume": {
-                "type": "string"
-              }
+          "properties": {
```

Comment:
did you check if it is breaking change for the frontend?

Reply:
it doesn't break anything